test/command/command.test.ts

```diff
@@ -525,6 +525,22 @@ describe('command', () => {
     })
     .it('json enabled/pass through disabled/--json flag before --/jsonEnabled() should be true')
 
+    fancy
+    .stdout()
+    .do(async () => {
+      class CMD extends Command {
+          static enableJsonFlag = true
+          async run() {}
+      }
+
+      // mock a scopedEnvVar being set to JSON
+      const cmd = new CMD([], {
+        bin: 'FOO', scopedEnvVar: (foo: string) => foo.includes('CONTENT_TYPE') ? 'json' : undefined,
```

Comment:
there's an `.env()` on fancy-test that lets you set env vars if you prefer

Reply:
that didn't work when I tried it